src/Model/PatientDictContainer.py

```diff
@@ -7,25 +12,60 @@
     num_points
     pixluts
 """
+from src.Model.Singleton import Singleton
+
 
+class PatientDictContainer(metaclass=Singleton):
 
-class PatientDictContainer:
+    def __init__(self):
+        # Initialize base requirements
+        self.path = None                # The path of the loaded directory.
+        self.dataset = None             # Dictionary of PyDicom dataset objects.
+        self.filepaths = None           # Dictionary of filepaths.
 
-    def __init__(self, path, dataset, filepaths, **kwargs):
+        self.additional_data = None     # Any additional values that are required (e.g. rois, raw_dvh, raw_contour, etc)
+
+    def set_base_values(self, path, dataset, filepaths, **kwargs):
```

Comment:
why "base"?
Are these the initial values?
Is this what gets called when loading a patient's data at the beginning?